sand/models/search.py

```diff
@@ -0,0 +1,23 @@
+from dataclasses import dataclass
+from typing import List
+from gena.serializer import get_dataclass_serializer
+
+
+@dataclass
+class SearchItem:
+    """
+    Search Item dataclass to save the values of each search item in a search
+    """
+    label: str
+    id: str
+    description: str
+    uri: str
+
+
+@dataclass
+class SearchPayload:
```

Comment:
why not let the interface return List[SearchItem]?

Reply:
Updated the return type and removed SearchPayload from the models